jfix-stdlib-concurrency/src/main/java/ru/fix/stdlib/concurrency/threads/ReschedulableScheduler.java

```diff
@@ -120,9 +144,13 @@ public void run() {
             //Preventing concurrent task launch for the case when
             // previously launched task still working,
             // schedule changed and triggered new scheduled task to launch
-            if(!taskIsRunning.compareAndSet(false, true)){
+            if (!taskIsRunning.compareAndSet(false, true)) {
+                log.trace("Preventing concurrent task launch; scheduledFuture={} with hash={}",
+                        scheduledFuture, System.identityHashCode(scheduledFuture));
                 return;
             }
+            log.trace("taskIsRunning flag is set to true; scheduledFuture={} with hash={}",
```

Comment:
seems like we do not need this log, if we will put two othrer logs in
- skip wrong invocations
and
- before task.run() method directly